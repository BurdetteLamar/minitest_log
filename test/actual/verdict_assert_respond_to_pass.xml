<log>
  <summary verdicts='2' failures='0' errors='0'/>
  <section name='Test' method='verdict_assert_respond_to?' type='pass'>
    <section name='Args=[&quot;&quot;, :size]'>
      <verdict method='verdict_assert_respond_to?' outcome='passed' id='0' message='Message 0'>
        <object class='String' value='&quot;&quot;'/>
        <method class='Symbol' value=':size'/>
      </verdict>
    </section>
    <section name='Args=[nil, :class]'>
      <verdict method='verdict_assert_respond_to?' outcome='passed' id='1' message='Message 1'>
        <object class='NilClass' value='nil'/>
        <method class='Symbol' value=':class'/>
      </verdict>
    </section>
  </section>
</log>
