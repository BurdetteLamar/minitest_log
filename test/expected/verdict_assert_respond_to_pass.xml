<log>
  <summary verdicts='3' failures='0' errors='0'/>
  <section name='Test' method='verdict_assert_respond_to?' type='pass'>
    <section name='Args=[&quot;&quot;, :size]'>
      <verdict method='verdict_assert_respond_to?' outcome='passed' id='0' message='Message 0'>
        <object class='String' value='&quot;&quot;'/>
        <method class='Symbol' value=':size'/>
      </verdict>
    </section>
    <section name='Args=[nil, :class]'>
      <verdict method='verdict_assert_respond_to?' outcome='passed' id='1' message='Message 1'>
        <object class='NilClass' value='nil'/>
        <method class='Symbol' value=':class'/>
      </verdict>
    </section>
  </section>
  <verdict method='verdict_assert_equal?' outcome='passed' id='__error_count__'>
    <expected class='Integer' value='0'/>
    <actual class='Integer' value='0'/>
  </verdict>
</log>
